ANSWERS.md

```diff
@@ -1,6 +1,11 @@
 ### Questions:
 1. What is the DOM?
+    Version of your html presented by your broswer
```

Comment:
Your answers were all correct, but I think it would help if you expanded on them a little more.